### ISSUE TITLE
Need new scheming presets specific to NDM

### ISSUE THREAD

**Issue by thriuin** [open-data/ckanext-stcndm#1]
There's a need for presets/snippets to handle:
-  multi-value semi-colon delimited strings
-  code values that display and index the code text but only save the code value in one field
-  looks up the list of code values and labels from a specified schema

**Comment by wardi** [comment]
I mentioned stealing and modifying the validators in https://github.com/open-data/ckanext-repeating/blob/master/ckanext/repeating/validators.py#L5 to accept the semicolon-delimited input and split it into a list on the update/create side.

Happy to help with this on Monday

**Comment by thriuin** [comment]
Absolutely. This is was more a reminder to myself. Also the looking up of choices for a drop-down list from existing dataset records (with a 'code' or 'taxonomy subject'  or whatever schema) would be a common pattern, so we'll probably want some kind of re-usable code. 

I have a draft schema for cubes so we can review that on Monday. Thanks, and have a great weekend.
